vault-core/src/main/java/org/apache/jackrabbit/vault/packaging/Version.java

```diff
@@ -135,44 +136,40 @@ public String[] getNormalizedSegments() {
     }
 
     /**
-     * Compares this version to the given one, segment by segment without any special
-     * "SNAPSHOT" handling.
+     * Compares this version to the given one. The comparison is compatible to the ordering used by
+     * <a href="https://maven.apache.org/">Apache Maven</a>. It version consists normally from 3 numbers -
+     * major version, minor version and patch level, and can be followed by a dash and a qualifier like SNAPSHOT.
+     * Version numbers can also consist of fewer or more numbers.
+     * If the comparison is not resolved by comparing the numbers, the algorith resorts to the qualifier - see
+     * <a href="https://cwiki.apache.org/confluence/display/MAVENOLD/Versioning">"Versioning" on Maven Wiki</a>
```

Comment:
This resource is really outdated, rather link something more recent like https://maven.apache.org/pom.html#version-order-specification